tests/ios/unit-test/BUILD.bazel

```diff
@@ -70,3 +70,13 @@ ios_unit_test(
     infoplists = [{"NSPrincipalClass": "TestSuiteObserver"}],
     minimum_os_version = "11.0",
 )
+
+ios_unit_test(
+    name = "ExplicitFeatures",
+    srcs = [
+        "empty.m",
+        "empty.swift",
+    ],
+    features = ["apple.skip_codesign_simulator_bundles"],
```

Comment:
Is this a performance optimization or just a test?

Reply:
Just a test.  This particular value just happened to be the one that led me to find the issue.